Slambook/index.html

```diff
@@ -1,51 +1,118 @@
-<!doctype html>
-<html lang="en">
```

Comment:
It is good practice to declare and initialize variables on separate lines to improve readability.